src/utils/getWindowDimensions.ts

```diff
@@ -0,0 +1,23 @@
+import { Dimensions } from 'react-native';
+import {getOrientation, getInitialOrientation} from 'react-native-orientation-locker';
+import { orientations } from '../redux/constants/orientationsConstants';
+
+
+const getWindowDimensions = () => {
+  const orientation = getOrientation ? getOrientation() : getInitialOrientation();
```

Comment:
This getOrientation() method accepts a callback function and doesn't return anything. It has no use here because it is returning undefind here. For now getInitialOroentation would fulfil the purpose I think